cypress/integration/articleSpec.js

```diff
@@ -8,7 +8,7 @@ import {
   shouldContainStyles,
 } from '../support/testHelper';
 
-describe('News Article', () => {
+describe('Article Body Tests', () => {
```

Comment:
We should also move https://github.com/bbc/simorgh/pull/387/files#diff-d8b08c81c1b0e31ef3e9bf8a801301e6R18 out of this spec file.